.vscode/launch.json

```diff
@@ -10,7 +10,8 @@
       "protocol": "inspector",
       "console": "internalConsole",
       "sourceMaps": true,
-      "outFiles": ["${workspaceRoot}/distribution"]
+      "outFiles": ["${workspaceRoot}/distribution"],
+      "runtimeExecutable": "/Users/joshua/.nvm/versions/node/v10.12.0/bin/node"
```

Comment:
I, uh, shouldn't have committed this. Do you mind plucking it out?

Reply:
:+1: